src/components/Common/Footer.jsx

```diff
@@ -0,0 +1,10 @@
+
+import React from 'react';
+
+export default class Footer extends React.Component {
+  render() {
+    return (
+      <footer>FOOTER LOCATION</footer>
+      );
+  }
+}
```

Comment:
Why not introduce a Template class that combines Header and Footer.

```
<Header />
{this.props.children}
<Footer />
```

Then you can consume it with code like this:
```
<Template>
    <h1>Something</h1>
    <p>Something else</p>
</Template>
```